src/utils/downloader.c

```diff
@@ -7774,8 +7821,20 @@ GF_Err gf_dm_sess_send(GF_DownloadSession *sess, u8 *data, u32 size)
 	}
 #endif
 
+	if (sess->should_holdback_data && !sess->holdback_data_complete) {
+		if (!data || !size)
+			sess->holdback_data_complete = GF_TRUE;
+
+		if (!sess->holdback_data_complete) {
+			sess->holdback_data_size += size;
+			sess->holdback_data = gf_realloc(sess->holdback_data, sess->holdback_data_size);
+			memcpy(sess->holdback_data + sess->holdback_data_size - size, data, size);
```

Comment:
I had to change here:

```
			memcpy((u8*) sess->holdback_data + sess->holdback_data_size - size, data, size);
```

Reply:
I assume you were getting segfaults otherwise?

Reply:
No, I just could not build it

Reply:
> 1>C:\Project\castlabs.com\gpac.io\gpac_public\src\utils\downloader.c(7831,31): error C2036: 'void *': unknown size
1>C:\Project\castlabs.com\gpac.io\gpac_public\src\utils\downloader.c(7831,58): error C2036: 'void *': unknown size

Reply:
My compiler didn't complain about this. I'll push this change though